test/forms/incomeSources/cease/CeaseIncomeSourceEndDateFormProviderSpec.scala

```diff
@@ -0,0 +1,281 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.incomeSources.cease
+
+import auth.MtdItUser
+import enums.IncomeSourceJourney.{ForeignProperty, IncomeSourceType, SelfEmployment, UkProperty}
+import forms.models.DateFormElement
+import models.admin.IncomeSourcesNewJourney
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.data.{Form, FormError}
+import play.api.i18n.Messages
+import services.{DateService, DateServiceInterface}
+import testConstants.BaseTestConstants.{testMtditid, testMtditid2, testNino, testSelfEmploymentId}
+import testConstants.incomeSources.IncomeSourceDetailsTestConstants.{businessIncome2, ukForeignSoleTraderIncomeSourceBeforeEarliestStartDate, ukPlusForeignPropertyWithSoleTraderIncomeSource}
+import testUtils.TestSupport
+import uk.gov.hmrc.auth.core.AffinityGroup.Individual
+
+import java.time.LocalDate
+import scala.language.postfixOps
+
+class CeaseIncomeSourceEndDateFormProviderSpec extends AnyWordSpec with Matchers with TestSupport {
+  val mockDateService: DateService = app.injector.instanceOf[DateService]
+  val ceaseEndDateForm: CeaseIncomeSourceEndDateFormProvider = app.injector.instanceOf[CeaseIncomeSourceEndDateFormProvider]
+
+  val testUser: MtdItUser[_] = MtdItUser(
```

Comment:
can we update testUser~ variables name to include more details what these users represents ?